project-progression-template.md

```diff
@@ -4,24 +4,47 @@
-16. List of project's official communication channels (slack, irc, mailing lists)
-17. Social media accounts
-18. Existing financial sponsorship
-19. Trademark status
-20. Preferred maturity level (Sandbox, Incubation, Graduation, or Emeritus)
-21. Any additional information the TAC and Board should take into consideration when reviewing your proposal.
+All fields required. Throughout this document, the short answers (280 characters or less) are required; extended answers are encouraged but optional.
+### Project Information
+1.  Current name of project and any public repos/websites, social media, or chat channels
+2.  Project point of contact (name and email address)
+3.  Current project license
+4.  If donated to the Consortium, will this project be licensed differently?
+5.  Does your project currently have a CLA? If so, please link to the documentation of your CLA.(Using the Developer Certificate of Origin is considered a CLA.)
+6.  Have you read the Consortium’s project acceptance process and understand the IP and trademark statuses of project’s donated to the Consortium? [LINK] (A “no” answer is okay; the committee can answer questions about the policy.)
+7.  Does the project currently have a code of conduct? (If so, please specify and or link to code of conduct page in your documentation)
+8.  Does the project currently have a vulnerability disclosure contact and process? (If so, please link)
+9.  Are any government entities currently providing funding that supports this project? If so, please list nature of funding.
+### Project Functionality
+10.  What does this project do?
+- 280 char summary:
+- Extended answer:
+11. What proprietary dependencies does this project have, if any? For example, it is an open source library that is used on a proprietary platform.
+### Project Fit
+12. What problem is this project solving, and how does that advance the CCC’s mission? For reference, the mission statement is: Confidential Computing is the protection of data in use by performing computation in a hardware-based Trusted Execution Environment. [Mission Statement](README.md)
+- 280 char summary:
+- Extended answer:
+13. Is the project currently compatible with, or does it depend upon or integrate with, existing CCC projects? (Note this is not a requirement and does not impact acceptance; this is just for information gathering) 
+14. Is the project used in production environments today that you are aware of?
+15. Are there currently any commercial offerings of this project?
+### Current contributors and contribution structures
+16. What companies or groups are currently contributing to this project? Do you have an approximate breakdown of contribution percentage by company/group affiliation over the past 6 months?
+17. What incentive or motivation is there for someone to be an external contributor to this project?
+- 280 char summary:
+- Extended answer:
+### Project Goals
+18. Why was this project open sourced (or intended to be open sourced)?
```

Comment:
Discussion: this is phrased about history, and the original reason may be obsolete now and not particularly material.  Suggest removing 18 and just keeping 19 about the current state.

Reply:
+1 to remove 18